.github/workflows/ci.yml

```diff
@@ -52,6 +70,11 @@ jobs:
         luarocks install luacov
         luarocks install luacov-reporter-lcov
 
+    - name: Install and run darklua
```

Comment:
we should be able to install darklua via foreman as of January, thanks to @jeparlefrancais adding gitlab support to foreman :)